spec/openapi.yaml

```diff
@@ -708,6 +708,10 @@ components:
           type: integer
           description: Estimated print time for the file in seconds.
           nullable: true
+        timezoneOffset:
```

Comment:
For this we need definitive answer from Marek Mosna.

@ondratu wants to show estimate end in the browser timezone.

Reply:
Marek Mosna also wants to send data according to https://docs.octoprint.org/en/master/api/datamodel.html#sec-api-datamodel-jobs-progress. So `printTimeLeft` in seconds and show the estimated print end in clients (browser) timezone